data.sql

```diff
@@ -1,5 +1,7 @@
-/* Populate database with sample data. */
+/* Populate database with vetinary data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animal (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (2, 'Gabumon', '2018-11-15', 2, TRUE, 8.0),
+(3, 'Pikachu', '2021-07-01', 1, FALSE, 15.04),
+(4, 'Devimon', '2017-05-17', 5, TRUE, 11),
+(5, 'Agumon', '2020-02-03', 0, TRUE, 10.23);
```

Comment:
- [ ] You are almost there! However, there should be 7 `animals` added to the database:

![Screen Shot 2022-08-10 at 8 43 19 AM](https://user-images.githubusercontent.com/83717285/183916651-7d9abf7b-3407-4de1-8073-026c6442319f.png)

It would be great to add the missing ones as required by the activity.

Reply:
Hi there @smunozmo Thank you for the review, I have added the extra data as suggested here!

All the best-K